src/version-2-1-0.md

```diff
@@ -0,0 +1,31 @@
+Date: 2016-03-09
+Title: Release 2.1.0
+Tags: news, release notes
+Slug: uniqush-push-2-1-0
+Category: news
+
+Please check out our [release notes] for version 2.1.0.
+
+We've added new APIs for listing the subscriptions of a subscriber and for
+listing the services that exist.
+
+Concurrency issues in ADM and APNS have been fixed. The APNS implementation
+has been changed to use a new, reliable worker pool implementation. This
+implementation has been running at production scale at if(we) serving push
```

Comment:
"production scale" is ambiguous, readers of uniqush blog wouldn't be familiar with company or its products.

The uptime and number of pushes *one process* was processing for the most popular Android app would probably be more relevant.